features/factory_firmware/.gitignore

```diff
@@ -0,0 +1,21 @@
+/.metadata/
```

Comment:
Actually those ignore should be in the root folder of the Repo. Can you please move it.